registry/auth/token/certificates.go

```diff
@@ -0,0 +1,92 @@
+package token
```

Comment:
I agree, I think we could rename it to `certificates_test.go` so it doesn't get compiled into the binary 🤔

Reply:
I don't think it would get compiled in anyway if its unused, its more about making sure that future developers don't think its an interface that needs to be supported.

Just not exporting anything would be a good start.

Reply:
> I don't think it would get compiled in anyway if its unused

I am intrigued to learn if this is the case given it would have been in a package 🤔 Unfortunately my Go compiler knowledge is lacking here.

Can defo unexport, but I really do think we should consider chucking it into a test file 🤔

Reply:
https://www.reddit.com/r/golang/comments/6vbh65/are_unused_functions_compiled/?rdt=58347

Removed by the linker apparently.